tox.ini

```diff
@@ -1,27 +1,18 @@
 [tox]
-skipsdist = True
-usedevelop = True
-
+# Necessary because we are using pyproject.toml
+isolated_build = true
```

Comment:
Question: is the `isolated_build` setting necessary? The [Upgrading to tox v4](https://tox.wiki/en/latest/upgrading.html#removed-tox-ini-keys) page says that it's on by default:
> `isolated_build` Isolated builds are now always used.

Reply:
I must've been on an old version of tox